Game/sources/BattleSim.cpp

```diff
@@ -2,27 +2,33 @@
 #include <QApplication>
 #include <QJsonDocument>
 #include <QDebug>
+#include <cmath>
 
 #include <headers/Entity.h>
 
 #include "../headers/BattleSim.h"
 
-BattleSim::BattleSim(Entity* player){
+BattleSim::BattleSim(Entity* player, int minimax_depth){
+  minimax_depth_ = minimax_depth;
+
   player_ = player;
 
   // make enemy
   QJsonObject entity_data;
   int enemy_level =  player->GetLevel() + ( qrand() % 5 - 2 );
+  //int enemy_level = 10;
   enemy_level = (enemy_level<1)?1:enemy_level;
   entity_data["level"] = enemy_level;
-  entity_data["max_health"] = enemy_level * 10;
-  entity_data["max_magic"] = enemy_level * 10;
+  entity_data["max_health"] = floor(log(enemy_level+1)/log(1.017)); // FIX FLOATING NUMBERS
```

Comment:
I forgot what this was doing exactly.

Reply:
This sets their max health depending on their level. I am going to make it not magic numbers but for not it's just hard coded. I can also fix the void issue.